src/platform/forms-system/src/js/review/ReviewCollapsibleChapter.jsx

```diff
@@ -373,71 +370,38 @@ class ReviewCollapsibleChapter extends React.Component {
     }, 0);
   };
 
-  scrollToPage = key => {
-    scroller.scrollTo(
-      `${key}ScrollElement`,
-      getScrollOptions({ offset: scrollOffset }),
-    );
-  };
-
   render() {
-    let pageContent = null;
-
     const chapterTitle = this.getChapterTitle(this.props.chapterFormConfig);
-
-    if (this.props.open) {
-      pageContent = this.getChapterContent(this.props);
-    }
-
-    const classes = classNames('usa-accordion-bordered', 'form-review-panel', {
-      'schemaform-review-chapter-error': this.props.hasUnviewedPages,
```

Comment:
- "form-review-panel" and "usa-accordion-bordered' seem to be used by a few apps for styling
- There is CSS for "schemaform-review-chapter-error" (and the associated icon) can be removed